Tests/js/views/services/assets/ez-navigationhubviewservice-tests.js

```diff
@@ -562,8 +562,16 @@ YUI.add('ez-navigationhubviewservice-tests', function (Y) {
             this.service.removeNavigationItem(identifier, zone);
 
             Y.Array.each(this.service.get(zone + "NavigationItems"), function (item) {
+                var itemIdentifier;
+
+                if (item.config) {
```

Comment:
I would rather test if `item` is an instance of `Y.eZ.NavigationItemSubtreeView`